python_ta/contracts/__init__.py

```diff
@@ -578,9 +580,9 @@ def _display_annotation(annotation: Any) -> str:
     'int'
     >>> _display_annotation(list[int])
     'list[int]'
-    >>> from typing import List
-    >>> _display_annotation(List[int])
-    'typing.List[int]'
+    >>> from typing import list
```

Comment:
This entire doctest example needs to be updated (note that you can, and should, run the doctest example manually to ensure it passes)

Reply:
from my understanding, the doctest is designed to demonstrate three different types of annotations, including one that uses from typing import List. Since the class already fails in Python 3.8 due to the list[int] syntax producing a type error, I have reverted my previous changes to avoid further issues.